docs/user-stories.md

```diff
@@ -5,22 +5,39 @@
 - User
 - Admin
 
-## Epic
-
-### User Stories
-
-As a user
-
-- I would like to see clear instructions about how I am going to recycle.
-- I would like to be able to find nearest recycling locations.
-- I would like to answer survey questons anonymously about if I did recycle today.
-- I'd like to see other people's answers as statistics.
-- I'd like to find envirmental friendly online stores.
-- I'd like to find environment-friendly alternatives of popular products and services.
-
-As an Admin
-
-- I'd like to be able to add online store links
-- I'd like to be able to add product and services
-
-Website that crates awarness for envirement and encorages to recycle.
+## As a user, I want to track my enivronemntal habits
+- As a user, I want to be able to see a list of habits to add.
+- As a user, I want to be able to add, remove and edit habits.
+- As a user, I want to be able to mark my habits as 'successful' or 'failed'.
+- As a user, I want to be able to rate my habits from easy to hard.
+- As a user, I want to be able to see the statistics of my habits.
+- As a user, I want to be able to see my habit/activity score.
+- As a user, I want to be able to receive a reminder/notification to do my habits.
+- As a user, I want to be able to answer survey questons anonymously about if I did recycle today.
+- As a user, I want to be able to see other people's answers as statistics.
```

Comment:
yeah, this brings new ideas to my mind, that users can compare themselves with their town and city.